src/pages/stablecoins.tsx

```diff
@@ -526,7 +526,7 @@ const StablecoinsPage = ({ markets, marketsHasError }) => {
                   </Text>
                   <Flex direction="column">
                     <Box>
-                      <ButtonLink mb={4} me={4} to="https://1inch.exchange">
+                      <ButtonLink mb={4} me={4} target="_blank" href="https://matcha.xyz/tokens/ethereum/0x6b175474e89094c44da98b954eedeac495271d0f?sellChain=1&sellAddress=0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee">
```

Comment:
`to` was giving a "deprecated" warning so I update the usage to `href`.

Reply:
@pettinarip do you remember what the nuance here was? i see our `Link` component accepts both

Reply:
@minimalsm Would you prefer I revert back to the original usage?